lint.json

```diff
@@ -28,7 +28,7 @@
     "no-unused-expressions": "off",
     "no-unused-vars": [
       "warn"
-    ]
+    ],
```

Comment:
BTW this kind of mistake should be checked by your IDE or whatever editor you are using.

Reply:
we can add https://github.com/paulvollmer/jsonlint to the `npm test` command to check the json file(s)